jwst/tweakreg/tweakreg_step.py

```diff
@@ -162,6 +166,34 @@ def process(self, input):
                       "containing one or more DataModels.", ) + e.args[1:]
             raise e
 
+        if self.separation <= _SQRT2 * self.tolerance:
+            self.log.error(
+                "Parameter 'separation' must be larger than 'tolerance' by at "
+                "least a factor of sqrt(2) to avoid source confusion."
+            )
+            for model in images:
+                model.meta.cal_step.tweakreg = "SKIPPED"
+                # Remove the attached catalogs
+                if hasattr(model, "catalog"):
+                    del model.catalog
```

Comment:
Is this attribute removal needed? `model.catalog` is not set until [later in this step](https://github.com/spacetelescope/jwst/blob/3df35ef10cbfda904a7bfee947d8fd5e87e2092d/jwst/tweakreg/tweakreg_step.py#L252) and is removed before the step finishes.

Reply:
you are right. It is a leftover from a now dropped commit.